test/unit/io/stream/parse_condition_test.cpp

```diff
@@ -136,7 +149,7 @@ TEST(parse_condition, is_not_msg)
 TEST(parse_condition, is_in_interval)
 {
     using namespace seqan3;
-    is_in_interval<'a', 'z'> cond;
+    auto cond = is_in_interval<'a', 'z'>;
```

Comment:
`constexpr` missing?

Reply:
it is not required and with optimisations it will likely result in the same thing, but it I will add it for consistency.